include/color_spinor_field.h

```diff
@@ -16,6 +16,9 @@ namespace quda {
 
   struct FullClover;
 
+  // Declaration in advance
```

Comment:
What's the point of this pre-declaration?  Why not have the definition here too?

Reply:
No good reason, moving it.